betfairlightweight/resources/bettingresources.py

```diff
@@ -1,7 +1,38 @@
+import re
+
 from .baseresource import BaseResource
 
+CAMEL_CASE_PATTERN = re.compile(r"(?<!^)(?=[A-Z])")
+
 
-class EventType:
+def camel_case_to_snake_case(camel_case_str: str) -> str:
```

Comment:
Move to utils next to the [opposite](https://github.com/betcode-org/betfair/blob/82b4edade5c5848ff29fbcbf8304a605af17e86a/betfairlightweight/utils.py#L55)?

Reply:
Done